deployment/setup-keycloak.sh

```
@@ -0,0 +1,138 @@
+    "implicitFlowEnabled": false,
+    "redirectUris": [
+      "http://localhost:3000/*",
+      "http://localhost:5173/*"
+    ],
+    "webOrigins": [
+      "http://localhost:3000",
+      "http://localhost:5173"
+    ],
+    "attributes": {
+      "access.token.lifespan": "300",
+      "client.secret.creation.time": "'$(date +%s)'"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)